<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDialog</class>
 <widget class="QDialog" name="EventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="dateLabel">
     <property name="font">
      <font>
       <family>한컴 말랑말랑</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Date: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="leftWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="filterLayout">
         <item>
          <widget class="QLabel" name="filterLabel">
           <property name="font">
            <font>
             <family>한컴 말랑말랑</family>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Show :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="filterCombo">
           <property name="font">
            <font>
             <family>한컴 말랑말랑</family>
            </font>
           </property>
           <property name="placeholderText">
            <string>일정 선택</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="eventsLabel">
         <property name="font">
          <font>
           <family>한컴 말랑말랑</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Events :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="eventsListWidget">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rightWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="detailsGroup">
         <property name="font">
          <font>
           <family>한컴 말랑말랑</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Event Details</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="titleLayout">
            <item>
             <widget class="QLabel" name="titleLabel">
              <property name="font">
               <font>
                <family>한컴 말랑말랑</family>
                <pointsize>10</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Title :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="titleEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>제목을 입력해주세요</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="descriptionLabel">
            <property name="font">
             <font>
              <family>한컴 말랑말랑</family>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Description :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="descriptionEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::Box</enum>
            </property>
            <property name="placeholderText">
             <string>내용을 입력해주세요</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="timeLayout">
            <item>
             <widget class="QLabel" name="timeLabel">
              <property name="font">
               <font>
                <family>한컴 말랑말랑</family>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Time :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit">
              <property name="font">
               <font>
                <family>한컴 말랑말랑</family>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="priorityLayout">
            <item>
             <widget class="QLabel" name="priorityLabel">
              <property name="font">
               <font>
                <family>한컴 말랑말랑</family>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Priority :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="priorityCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="categoryLayout">
            <item>
             <widget class="QLabel" name="categoryLabel">
              <property name="font">
               <font>
                <family>한컴 말랑말랑</family>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Category :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="categoryCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="editButton">
            <property name="font">
             <font>
              <family>한컴 말랑말랑</family>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Edit</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="horizontalGroupBox">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="addButton">
               <property name="font">
                <font>
                 <family>한컴 말랑말랑</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
               <property name="icon">
                <iconset theme="QIcon::ThemeIcon::ListAdd"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteButton">
               <property name="font">
                <font>
                 <family>한컴 말랑말랑</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
               <property name="icon">
                <iconset theme="QIcon::ThemeIcon::EditDelete"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="font">
        <font>
         <family>한컴 말랑말랑</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="font">
        <font>
         <family>한컴 말랑말랑</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>EventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>EventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eventsListWidget</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>eventsListWidget</receiver>
   <slot>doItemsLayout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
